flink-connector-aws/flink-connector-dynamodb/src/test/java/org/apache/flink/connector/dynamodb/table/RowDataToAttributeValueConverterTest.java

```diff
@@ -547,12 +552,132 @@ void testInstantArray() {
         assertThat(actualResult).containsAllEntriesOf(expectedResult);
     }
 
+    @Test
```

Comment:
Could we add tests for delete with no primary key and verify exception

Reply:
Yes, great idea, I've done this now in latest commits!